assets/variant-list.js

```diff
@@ -223,6 +223,8 @@ class VariantList extends HTMLElement {
         const items = document.querySelectorAll('.variant-item');
 
         if (parsedState.description || parsedState.errors) {
+          const variantItem = document.querySelector(`[id^="Variant-${id}"] .variant-item__totals.small-hide .loading-overlay`);
+          variantItem.classList.add('loading-overlay--error')
```

Comment:
We add `loading-overlay--error` class here, but it's never removed. Is that ok?

Reply:
We dont need to because when something gets added/removed to cart the whole row gets re-rendered (no error class by default). And if there is no error, no error class is added